### PR TITLE
added new endpoints to system_control client

### PR DESCRIPTION
**Pull request by Vertiq-Ben**
- Added new endpoints to system_control client

### CODE REVIEW

**Comment by iq-fred** [comment]
This is the Github twin of of a Bibucket PR, I'll approve it once the Bitbucket one is fully approved so that we don't go back and forth

**Comment by Vertiq-Ben** [comment]
> This is the Github twin of of a Bibucket PR, I'll approve it once the Bitbucket one is fully approved so that we don't go back and forth

I merged the PR on bitbucket

**Review by iq-fred — APPROVED** [review]

**Pull request merged**
